spec/lib/insert/insert_donation_spec.rb

```diff
@@ -15,6 +15,7 @@
     describe 'param validation' do
       before(:each) do
         expect(Houdini.event_publisher).to_not receive(:announce).with(:donation_created,any_args)
+        expect(Houdini.event_publisher).to_not receive(:announce).with(:transaction_created,any_args)
```

Comment:
What do you think about moving those expects out from `before` and placing them into `it`s?

Reply:
I'm fine with that in theory. How would we avoid having to repeat those expects in every it though?